fss_check/check_fss.py

```diff
@@ -0,0 +1,503 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import logging
+from typing import Optional
+
+import kadi.events.query  # noqa: F401 # For kadi.events.models import (django setup)
+import matplotlib.pyplot as plt
+import matplotlib.style
+import numpy as np
+from astropy.table import Table
+from cheta.utils import logical_intervals
+from cxotime import CxoTime, CxoTimeLike
+from kadi import events
+from kadi.events.models import fuzz_states
+from Ska.Matplotlib import cxctime2plotdate, plot_cxctime, set_min_axis_range
```

Comment:
Super tiny suggestion, but ska_matplotlib for "new" code.

Reply:
Done.